kustomize/commands/edit/remove/removeconfigmap_test.go

```diff
@@ -1,15 +1,16 @@
-// Copyright 2019 The Kubernetes Authors.
+// Copyright 2022 The Kubernetes Authors.
 // SPDX-License-Identifier: Apache-2.0
 
 package remove //nolint:testpackage
 
 import (
 	"fmt"
-	"strings"
 	"testing"
 
+	"github.com/stretchr/testify/require"
+
 	"github.com/stretchr/testify/assert"
```

Comment:
nit: Do we need to have both `require` and `assert`? Would it be appropriate to replace all the `asserts` with `require`?

Reply:
Yes, I think it's fair to use require since it's just one single assertion.

Reply:
Updated!